db.py

```diff
@@ -96,7 +96,7 @@ def main():
     if command not in ["validate", "normalize"]:
         usage()
     else:
-        input = open("db.json", "r").read()
+        input = open("db.json", "r", encoding="utf-8").read()
```

Comment:
Nit: shouldn't line 18 be changed as well then?

Reply:
Yeah, I guess I just didn't test that codepath. Will do.